CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- `FullyQualifiedImportCheck` rule, wich analyzes if imports are fully qualified
```

Comment:
To make it more regular with our other changelog entries, how about:

```suggestion
- `FullyQualifiedImportCheck` analysis rule, which flags non-fully qualified imports.
```

Reply:
Done!